spec/models/comment_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before :all do
+    @user = User.create(name: 'Comment spec')
+    @post = Post.create(author: @user, title: 'Post Title')
+  end
+
+  after :all do
+    @user.destroy
+    @post.destroy
+  end
+
+  context '#create instance' do
+    it 'User and Post should be valid' do
+      expect(Comment.new(user: @user, post: @post)).to be_valid
+    end
+
+    it 'check to create comment without a post to the user' do
+      expect(Comment.new(user: @user)).to_not be_valid
+    end
+
+    it 'check to create comment to a post without a user' do
+      expect(Comment.new(post: @post)).to_not be_valid
+    end
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=equal%20to%20zero.-,Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.,-Need%20a%20big), Please add unit tests for the `update_comments_counter` method.